src/Symfony/Component/Dotenv/Command/DebugCommand.php

```diff
@@ -125,11 +119,7 @@ private function getEnvFiles(): array
             $files[] = '.env.local';
         }
 
-        if (!is_file($this->getFilePath('.env')) && is_file($this->getFilePath('.env.dist'))) {
-            $files[] = '.env.dist';
-        } else {
-            $files[] = '.env';
-        }
+        $files[] = !is_file($this->getFilePath('.env')) && is_file($this->getFilePath('.env.dist')) ? '.env.dist' : '.env';
```

Comment:
This does not add anything and I would argue that it makes it harder to read. In general, not worth the change (and time of review and merge) imho